### DISCUSSION TITLE
Plex Tokens

### DISCUSSION THREAD

**Discussion by JacquelineHyde** [Taxel/PlexTraktSync#1288] [Support] [answered]
What is the difference between these token settings in the .env file? I'm getting other people's stuff synced to my Trakt account.

PLEX_TOKEN=
PLEX_OWNER_TOKEN=
PLEX_ACCOUNT_TOKEN=

**Comment by simonc56** [comment, marked as answer]
__PLEX_TOKEN__
The token granting user access to the chosen PMS. For the PMS admin, it is the same token to access PMS and also the user's Plex online account.
For a user who don't own the PMS, it gives only access to PMS libraries.

__PLEX_OWNER_TOKEN__
The token of the PMS administrator/owner. This is stored/used only for __managed users__ (home users) so the script can get their watchlist from Plex online servers. Managed users don't have a Plex account, they're local users.

__PLEX_ACCOUNT_TOKEN__
The account token of the Plex user who have a Plex account but don't own the PMS (use it as shared library). This is stored/used only if user __uses a shared PMS__. Needed to fetch its watchlist from Plex online servers.

**Comment by SawkeeReemo** [comment]
So to sync a managed user, what do you do? I'm guessing from this explanation you only need the PLEX_OWNER_TOKEN filled in along with the managed user's profile name and the server name, yes? I think I'm on version [0.27.11](https://github.com/Taxel/PlexTraktSync/releases/tag/0.27.11), I just pulled `latest` on the docker pull.

**Reply by simonc56** [reply]
The script will automatically fetch and save needed tokens during setup.
You don't need to manage them.
To start setup from scratch, you can remove the 3 files where tokens are stored :
`servers.yml`
`.env`
`.pytrakt.json` (for trakt tokens)

**Reply by SawkeeReemo** [reply]
[I’m going to link to my other discussion here because I’ve offered more info there.](https://github.com/Taxel/PlexTraktSync/discussions/1629) In short: I ran into a bug where after I selected my server, the script crashes saying that the server did not exist. Then once I ran plextraktsync again, it ran fine but never asked me to choose a user. 

Also, I just realized I didn’t give you specs:
Synology DSM 7.2
plextraktsync in a Docker container
Plex (beta PMS) is running inside DSM, and is not currently in a Docker container
